PyDBtickets/TrainTicketDocument.py

```diff
@@ -182,15 +162,19 @@ def update_cost_sheet(self, sheet_file_path):
         u'Gross',
         u'Ust In / Buchungsmonat'
         """
+        #todo: add id here.
+        #todo: make this idempotent idempotent
         update_row = ["bahn",
-                      u"_".join(self.from_to) +
+                      "_".join(self.from_to) +
                       self.travel_date.strftime("%Y%m%d"),
                       self.date_bought.strftime("%Y-%m-%d"),
-                      self.gross_price - self.vat,
-                      self.vat,
+                      self.gross_price - self.payed_vat,
+                      self.payed_vat,
                       self.gross_price,
-                      self.date_bought.strftime("%Y-%m")
+                      self.date_bought.strftime("%Y-%m"),
+                      self.filename.rstrip('.pdf').split('/')[-1]
                      ]
         sheet = pyexcel.get_sheet(file_name=sheet_file_path)
-        sheet.row += update_row
-        sheet.save_as(sheet_file_path, dest_encoding="utf-8")
+        if not update_row in sheet.row:
```

Comment:
ah I see, isn't this sufficent for idempotency? what's the ToDo in lines 165/166?   
`if a not in b` is more pythonic than `if not a in b`